dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -274,12 +274,21 @@ object Datadog {
 
     internal const val MESSAGE_ALREADY_INITIALIZED =
         "The Datadog library has already been initialized."
-    internal const val MESSAGE_NOT_INITIALIZED = "Datadog has not been initialized.\n" +
-            "Please add the following code in your application's onCreate() method:\n" +
-            "Datadog.initialize(context, \"<CLIENT_TOKEN>\");"
+    internal const val MESSAGE_NOT_INITIALIZED = "Datadog has not been " +
+        "correctly initialized.\n" +
+        "Please add the following code in your application's onCreate() method:\n" +
+        "val config = DatadogConfig" +
+        ".Builder(\"<CLIENT_TOKEN>\", \"<ENVIRONMENT>\").build()\n" +
+        "Datadog.initialize(context, config);"
```

Comment:
You don't need the `correctly` adverb here. This message happens because the library is not initialized at all.